website/client/components/challenges/challengeModal.vue

```diff
@@ -243,7 +243,7 @@ export default {
     },
   },
   computed: {
-    ...mapState({user: 'user.data', party: 'party'}),
+    ...mapState({user: 'user.data', party: 'party.members.data'}),
```

Comment:
I'm confused as to why the party data is inside `party.members.data`?

Reply:
Me too. Looking at ```party:fetchMembers```, I'd expect the API response to be plugged into ```store.state.party```. But when you log out the contents of the store, that is where the data is. I haven't changed anything about how this data was stored. I've looked around and haven't seen anywhere else yet that could be changing it.

Reply:
`party:fetchMembers` has `party.members` as the `path` option so it's normal that the data ends up there but we're talking about the party members profile data not the party group data (chat, leader, description, ...)